test/sql/loader.sql.in

```diff
@@ -208,6 +208,11 @@ SET force_parallel_mode = 'on';
 SET max_parallel_workers_per_gather = 1;
 SELECT count(*) FROM test;
 
+CREATE EXTENSION timescaledb_osm VERSION 'mock-1';
+
+-- Test that OSM process utility hook works:  it should see this DROP TABLE.
+DROP TABLE test;
+
```

Comment:
we also need a test with disable.load for OSM.

Reply:
As discussed yesterday, working on a follow-up for this.

@cevian, Gayathri and I think this is ready to go as is.  Thanks!